cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -82,36 +111,38 @@ metadata:
   name: token-${safe_username}
 type: Opaque
 EOF
-  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" &
+  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" "${SYSTEM_NAMESPACE}" &
```

Comment:
Sorry, I should have been specific - I was responding to a GitHub line note. It has a tendency to leave comments if the code hasn't changed. In this case, the code didn't change, but the surrounding code did change, and the line note was no longer applicable. I was merely documenting for posterity that anyone that happened to be looking at the GitHub default diff view didn't have to pay attention to @marekbiskup's comment, and why.